auth/src/test/java/io/grpc/auth/GoogleAuthLibraryCallCredentialsTest.java

```diff
@@ -206,6 +208,23 @@ public void credentialsFailsWithIoException() throws Exception {
     assertEquals(exception, status.getCause());
   }
 
+  @Test
+  public void credentialsFailsWithUnretryableRetryableException() throws Exception {
```

Comment:
For complete test coverage can we have another test where the `Exception` is not a `RetryableException` (but an `IOException` or not even an `IOException`) and verify that the code is `UNAUTHENTICATED` ?

Reply:
See credentialsFailsWithRuntimeException

Reply:
yes!